blocks/form/form.js

```diff
@@ -20,7 +20,7 @@ function loadScript(url) {
 }
 
 function constructPayload(form) {
-  const payload = {};
+  const payload = { submissionDate: (new Date()).toUTCString() };
```

Comment:
Is it expected UTC string?

Reply:
I added this based on my understanding, I have asked the customer https://adobe-dx-support.slack.com/archives/C04T6MD0TDM/p1689250277874679 
Thank you